enhancements/network/sdn-live-migration.md

```diff
@@ -0,0 +1,442 @@
+   rather than using namespace VNIDs, since packets from OVN nodes will have the
+   VNID 0 set.
+2. To be compatible with Windows node VXLAN implementation, OVN Kubernetes
+   hybrid overlay uses the peer host interface MAC as the VXLAN inner dest MAC
+   for egress traffic. Therefore, when packets arrive at the br0 of the SDN
+   node, they cannot be forwarded to the pod interface correctly. We need to add
+   a flow to change the dst MAC to the pod interface MAC for such ingress
+   traffic.
+
+### MTU Consideration
+
+There are two related but distinct MTU values to consider: the hardware MTU and
+the cluster MTU. The cluster MTU is the MTU value for pod interfaces. It is
+always less than your hardware MTU to account for the cluster network overlay
+overhead. The overhead is 100 bytes for OVN Kubernetes and 50 bytes for SDN. For
+example, if the hardware MTU is 1500, then the auto-probed cluster MTU would be
+1400 for OVN Kubernetes and 1450 for OpenShift SDN. During the migration, this
+MTU mismatch would break some cross-CNI traffic.
+
+To avoid this issue, before starting the migration, users need to decrease the
+cluster MTU by 50 bytes following this instruction:
+[changing-cluster-network-mtu](https://docs.openshift.com/container-platform/4.13/networking/changing-cluster-network-mtu.html).
+
+### The Traffic Path
+
+#### Packets going from OpenShift SDN to OVN Kubernetes
+
+On the SDN side, it doesn't need to know if the peer node is a SDN node or an
+OVN node. We reuse the existing VXLAN tunnel rules on the SDN side.
+- Egress NetworkPolicy rules and service proxy happen as normal.
+- When the packet reaches table 90, it will hit a "send via vxlan" rule that was
+  generated based on a HostSubnet object.
+
+On the OVN side:
+- OVN accepts the packet via the VXLAN tunnel, ignores the VNID set by SDN, and
+  then just routes it normally.
+- Ingress NetworkPolicy processing will happen when the packet reaches the
+  destination pod's switch port, just like normal.
+  - Our NetworkPolicy rules are all based on IP addresses, not "logical input
+    port", etc., so it doesn't matter that the packets came from outside OVN and
+    have no useful OVN metadata.
+
+#### Packets going from OVN Kubernetes to OpenShift SDN
+
+On the OVN side:
+- The packet just follows the same path as the hybrid overlay.
+
+On the SDN side:
+- We have to change ingress NetworkPolicy processing to be based entirely on pod
+  IPs rather than using namespace VNIDs since packets from OVN nodes won't have
+  the VNID set. There is already code to generate the rules that way, though,
+  because egress NetworkPolicy already works that way.
+
+### Workflow Description
+
+1. The admin kicks off the migration process by updating the custom resource
+   `network.config`:
+   
+   ```bash
+   $ oc patch Network.config.openshift.io cluster --type='merge' --patch '{"metadata":{"annotations":{"network.openshift.io/live-migration":""}},"spec":{"networkType":"OVNKubernetes"}}'
```

Comment:
Should the annotation read as `network-type-live-migration` and statuses read as `NetworkTypeLiveMigration...` so it is clear it is about network type migration? Perhaps they could be condensed and remove the `live` part of it if there is no confusion possible with the offline migration since that one does not have similar annotations or statuses.